app/views/templates/playlists.jst

```diff
@@ -0,0 +1,12 @@
+<h4>YOUR MUSIC</h4>
```

Comment:
Please avoid uppercase in your markup, you can transform it in CSS if you really need it.

Think about localization too: use token strings ready for your localization lib in your templates.